nc/tests/test_api.py

```diff
@@ -23,7 +23,7 @@ def test_list_agencies(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         # Other Agencies may have been left around from other tests
         self.assertIn((agency.pk, agency.name), [
-            (a.pk, a.name) for a in Agency.objects.all()
+            (a['id'], a['name']) for a in response.data
         ])
 
```

Comment:
Can we add a comment, maybe at the top of this test case, that the api treats "Hispanic" as a race, even though it's stored in the `ethnicity` field in the database?